node/service/src/lib.rs

```diff
@@ -712,6 +718,18 @@ where
 		Some(backoff)
 	};
 
+	// For now BEEFY is explicitly disabled on non-test chains.
+	if config.chain_spec.is_polkadot() ||
+		config.chain_spec.is_kusama() ||
+		config.chain_spec.is_westend()
+	{
+		gum::warn!(
+			"Refusing to enable BEEFY on a production network; \
+                   BEEFY is still experimental."
+		);
+		enable_beefy = false;
```

Comment:
Do we want to force disable it or just warn and let the user decide? If we want to prevent that for the user anyways, I'd say this is an arg parse consistency issue and should error quickly rather than auto-disable and move on.